packages/core/content-manager/server/src/services/document-manager.ts

```diff
@@ -85,7 +85,7 @@ const documentManager = ({ strapi }: { strapi: Core.Strapi }) => {
 
     async create(uid: UID.CollectionType, opts: DocServiceParams<'create'> = {} as any) {
       const populate = opts.populate ?? (await buildDeepPopulate(uid));
-      const params = { ...opts, status: 'draft' as const, populate };
+      const params = { ...opts, status: opts?.status ?? ('draft' as const), populate };
```

Comment:
Probably the same as :

```
const params = { ...opts, status: opts?.status, populate };
```

While this is valid, we are allowint to create a draft and publish it, which I don't think is what the document manager is intended to do atm.

This might have other security implications, e.g user with only create permissions (not publish) could modify the request to add "status": "publish" and publish a document